### DISCUSSION TITLE
I have a new Quarto Pub.  In settings, I see "danger zone".

### DISCUSSION THREAD

**Discussion by hueyrb** [quarto-dev/quarto-cli#6823] [Q&A]
### Description

Yesterday I uploaded a website onto Quarto Pub.
This morning in settings, I'm seeing "Danger zone" & "Delete site" options.
Is this a concern?

**Comment by cderv** [comment]
This options are there for any publish content in the owner account. Other user can't see those - only the owner should see that.

**Reply by hueyrb** [reply]
Thanks!  I was concerned I might have been hacked.
